hudi-cli/src/main/java/org/apache/hudi/cli/HoodieCLI.java

```diff
@@ -74,7 +74,8 @@ public static void initFS(boolean force) throws IOException {
   }
 
   public static void refreshTableMetadata() {
-    setTableMetaClient(new HoodieTableMetaClient(HoodieCLI.conf, basePath, false, HoodieCLI.consistencyGuardConfig));
+    setTableMetaClient(new HoodieTableMetaClient(HoodieCLI.conf, basePath, false, HoodieCLI.consistencyGuardConfig,
```

Comment:
Can we use an overloaded constructor for HoodieTableMetaClient and avoid passing null ?

Reply:
Made the changes @n3nash